configuration/secrets/env_blank.json

```diff
@@ -1,5 +1,6 @@
 {
   "Logging Level": "INFO",
+  "Logging Directory": "",
```

Comment:
I know it's not required, but typically spaces aren't used in identifier names.  Underscores or camelCase would be used instead.  Apologies for not mentioning this earlier.

Reply:
@lsloan, thanks for the tip, I just fixed this in my latest commit.